scripts/get_qemu.sh

```diff
@@ -1,30 +1,35 @@
 #!/usr/bin/env bash
```

Comment:
Delete this file as it's no longer used?

Reply:
Hello
I've modified the script to use `qemu-user-static` instead of  `qemu-user-static:register`.
> `multiarch/qemu-user-static` image container includes both a register script to register `binfmt_misc` entries and all the `/usr/bin/qemu-$arch-static` binary files in the container in it.

So yes if we keep this solution it will not be needed anymore.

Reply:
When the call to register is made and `qemu-*-static` binaries will be registered and loaded in memory. So there is no need to copy them _manually_ on the host. Here is a quote from the project [qus](https://github.com/dbhi/qus) which inspired this new design as stated in this https://github.com/multiarch/qemu-user-static/pull/75.

> The required `qemu-*-static` binaries (which are all included in the image) will be loaded and registered. The container will then exit. From there on, images and/or binaries for foreign architectures can be executed.

Reply:
One improvement could be to use one of the `multiarch/qemu-user-static:$from_arch-$to_arch` images instead of the image containing all the binaries to download and load only those required.